### ISSUE TITLE
Research Composite Indicators for 15 Minute Score, Livability Indicator

### ISSUE THREAD

**Issue by Philipp-Otter** [goat-community/goat#1890]
Investigate whether local accessibility indices or subscores should be standardized or quantified prior to the aggregation.

- indicator formalism in the context of spatial data analysis
- geospatial indicators
- environmental indicators
- guide for composite indicator construction published by the Organisation for Economic Co-operation and Development  (OECD) 
- Mazziota–Pareto Index (MPI) method
- Composite measure (https://en.wikipedia.org/wiki/Composite_measure)
- Index (statistics) https://en.wikipedia.org/wiki/Index_(statistics)
- visual vocabulary financial times
- Handbook on Constructing Composite Indicators METHODOLOGY AND USER GUIDE (https://www.oecd.org/sdd/42495745.pdf)
- https://www.youtube.com/watch?v=dRn-inPyy54
![image](https://user-images.githubusercontent.com/91256148/217592615-ff47b1be-198f-4c2e-a71e-54a8fe038c18.png)
- other livability indicator (also composite indicator; https://link-springer-com.eaccess.tum.edu/article/10.1186/s12942-019-0178-8)
- normalization -> categorical scales
- monoton transformations
- panel normalization
- aggregate (composite) index of individual indicators
